pysmt/operators.py

```diff
@@ -55,7 +55,11 @@
 BV_ASHR,                                    # Arithmetic shift right (47)
 ARRAY_SELECT,                               # Array Select (48)
 ARRAY_STORE,                                # Array Store (49)
-ARRAY_VALUE                                 # Array Value (50)
+ARRAY_VALUE,                                # Array Value (50)
+
+DIV,                                        # Arithmetic Division (51)
```

Comment:
Ok, I think this boils down to 2 things:

1) Support for NIA
2) Printing

If we support NIA (that I think is reasonable), then we need to understand what it means to write ```2/x``` and whether we can write ```x^-c``` (negative exponent). From a quick test, it looks like Z3 can solve the first one without problems, while not the second one.

For printing, I think that x/y is more readable than (x * (y ^-1)).